src/tindev-results.html

```diff
@@ -99,64 +102,66 @@
           results: {
             type: Array,
             value: function() {
-              return [{
-                id: 123,
-                name: "GijsWeterings",
-                country_code: "NL",
-                languages: ["English", "Dutch"],
-                organizations: [
-                  {
-                    id: 999,
-                    name: "Tindev",
-                    url: "https://github.com/tindev"
-                  }
-                ],
-                projects: [
-                  {
-                    id: 888,
-                    name: "tindev-site",
-                    language: "Dutch",
-                    languages: ["Dutch", "English"],
-                    url: "https://github.com/tindev/tindev-site",
-                    issues: [],
-                    pull_requests: [{
-                      id: 630,
-                      url: "https://github.com/tindev/tindev-site/pull/1",
-                      title: "Add profiles of remaining teammembers",
-                      description: "- Add profiles for remaining team members\n- Change name from `Tinder for Bananas` to `Tindev`\n\ncloses #2 "
-                    }]
-                  }
-                ]
-              },
-              {
-                id: 124,
-                name: "TimvdLippe",
-                country_code: "NL",
-                languages: ["English", "Dutch"],
-                organizations: [
-                  {
-                    id: 999,
-                    name: "Tindev",
-                    url: "https://github.com/tindev"
-                  },
-                  {
-                    id: 999,
-                    name: "Tindev",
-                    url: "https://github.com/tindev"
-                  }
-                ],
-                projects: [
-                  {
-                    id: 888,
-                    name: "tindev-site",
-                    language: "Dutch",
-                    languages: ["Dutch", "English"],
-                    url: "https://github.com/tindev/tindev-site",
-                    issues: [],
-                    pull_requests: []
-                  }
-                ]
-              }];
+              return [
```

Comment:
why comment it out instead of deleting? we can get rid of the entire `value` property